src/librustdoc/html/render.rs

```diff
@@ -1289,8 +1289,9 @@ fn settings(root_path: &str, suffix: &str) -> String {
             .into(),
         ("auto-hide-attributes", "Auto-hide item attributes.", true).into(),
```

Comment:
overall i think we might need to fix up the language used for all these settings in a separate issue, it seems very unclear (cc @jyn514 thoughts?)

Reply:
It's not terribly clear but it does say exactly what it does (`attributes` is the technical term for `#[some_macro]`, right?). I think it would be helpful to have screenshots before and after, then it's very clear to see.

Reply:
I'm thinking of the entire set of names, not just this one, though. This one is kinda fine.